### ISSUE TITLE
Fractured foot bones

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5889]
**Preferred term label:** Fractured foot bones

**Synonyms** Metatarsal fracture; Broken foot

**Definition (free text, please give PubMed ID)** Fracture of the foot., PMID: 30847497

**Parent term (use hpo.jax.org/app)** Abnormality of the foot HP:0001760; Bone fracture HP:0020110

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
